java/operator/org.eclipse.theia.cloud.operator/src/main/java/org/eclipse/theia/cloud/operator/TheiaCloudOperatorLauncher.java

```diff
@@ -0,0 +1,29 @@
+/********************************************************************************
+ * Copyright (C) 2022 EclipseSource, Lockular, Ericsson, STMicroelectronics and 
+ * others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+package org.eclipse.theia.cloud.operator;
+
+import org.eclipse.theia.cloud.operator.di.AbstractTheiaCloudOperatorModule;
+
+public interface TheiaCloudOperatorLauncher {
+
+    public abstract void runMain(String[] args) throws InterruptedException;
```

Comment:
Modifier `public` is redundant on interface methods.

```suggestion
    abstract void runMain(String[] args) throws InterruptedException;
```